### ISSUE TITLE
`node-fetch` is no longer required

### ISSUE THREAD

**Issue by RobinTail** [arethetypeswrong/arethetypeswrong.github.io#79]
As of Node 18, `fetch` is within the globals:

https://dev.to/cloudx/nodejs-18-fetch-api-test-runner-module-and-more-2ckg

`@types/node-fetch` or `lib: ["dom"]` in `tsconfig.json`, however, might be required due to this issue (it's still experimental):

https://github.com/DefinitelyTyped/DefinitelyTyped/issues/60924

Nevertheless, your amazing package requires Node 18 via one of its dependencies anyway:

```
marked@5.1.0: The engine "node" is incompatible with this module. Expected version ">= 18". Got "16.20.0"

=> Found "marked@5.1.0"
info Reasons this module exists
   - "@arethetypeswrong#cli" depends on it
   - Hoisted from "@arethetypeswrong#cli#marked"
```

, so I'd suggest to set 

```
"engines": {
    "node": ">=18"
},
```

in `package.json` and remove `node-fetch` and `fetch-ponyfill` from the dependencies.

**Comment by RobinTail** [comment]
Hello @andrewbranch ,

Your package can not be installed in environment having Node 16 (at least without `--ignore-engines` flag of `yarn`).

Here is the output of my CI workflow [when I tried](https://github.com/RobinTail/express-zod-api/pull/1022/commits/f03bfed656f8dbd4b84ed7bfca3dad196e598f90):

```
### Run yarn install
yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error marked@5.1.0: The engine "node" is incompatible with this module. Expected version ">= 18". Got "16.20.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Process completed with exit code 1.
```

Currently I'm going to drop Node 16 in my library, so I'm beginning using your package in my CI.

**Comment by RobinTail** [comment]
Therefore, @andrewbranch , in order to support Node 16, I'd suggest to adjust the version of `marked` in your `package.json`:

```diff
- "marked": "^5.1.0"
+ "marked": "^5.1.2"
```

**Comment by RobinTail** [comment]
Perhaps I had this issue when `marked` 5.1.2 was not yet released, so it resolved to 5.1.0 when I tried to install your package.

**Comment by RobinTail** [comment]
🎉
